daemon/create_unix.go

```diff
@@ -60,14 +61,24 @@ func (daemon *Daemon) createContainerPlatformSpecificSettings(container *contain
 			return err
 		}
 
-		// never attempt to copy existing content in a container FS to a shared volume
-		if v.DriverName() == volume.DefaultDriverName {
```

Comment:
It's not, see `populateVolumes` below, which is what replaced this call.
There's also a test (already part of the codebase) which ensures this.